Classes/Command/UserCommandController.php

```diff
@@ -0,0 +1,71 @@
+<?php
+namespace Helhum\Typo3Console\Command;
+
+/*
+ * This file is part of the TYPO3 console project.
+ *
+ * It is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License, either version 2
+ * of the License, or any later version.
+ *
+ * For the full copyright and license information, please read
+ * LICENSE file that was distributed with this source code.
+ *
+ */
+
+use Helhum\Typo3Console\Mvc\Controller\CommandController;
+
+/**
+ * Commands for Backend users
+ */
+class UserCommandController extends CommandController {
+
+	/**
+	 * Create backend admin user
+	 *
+	 * @param string $username The username
+	 * @param string $password The password
+	 * @param string $realName Full Name
+	 * @param string $email E-Mail
+	 *
+	 */
+	public function createadminCommand($username, $password, $realName = '', $email = '') {
+		$userExists = $GLOBALS['TYPO3_DB']->exec_SELECTcountRows(
```

Comment:
please properly name the variables. This is rather `$userCount` than `$userExists` (which suggests the value being a boolean)